packages/manager/modules/account-migration/src/notification/notification.controller.js

```diff
@@ -0,0 +1,76 @@
+import { FAQ_LINK } from '../constants';
+
+export default class MigrationNotificationController {
+  /* @ngInject */
+  constructor(
+    $q,
+    $translate,
+    $state,
+    atInternet,
+    accountMigrationService,
+    OvhApiMe,
+    RedirectionService,
+  ) {
+    this.$q = $q;
+    this.$state = $state;
+    this.$translate = $translate;
+
+    this.atInternet = atInternet;
+    this.accountMigrationService = accountMigrationService;
+    this.OvhApiMe = OvhApiMe;
+    this.RedirectionService = RedirectionService;
+
+    this.AGREEMENTS_URL = this.RedirectionService.getURL('agreements');
+    this.CONTACTS_URL = this.RedirectionService.getURL('contacts');
+    this.ORDERS_URL = this.RedirectionService.getURL('orders');
+
+    this.migrationDetail = null;
+    this.needMigration = false;
+  }
+
+  $onInit() {
+    return this.$q
+      .all([
+        this.accountMigrationService.getMigrationList(),
+        this.accountMigrationService.getFormattedDates(),
+      ])
```

Comment:
Why not use an object instead ? 
```suggestion
      .all(({
        [migration]: this.accountMigrationService.getMigrationList(),
        migrationDates: this.accountMigrationService.getFormattedDates(),
     })
```

Reply:
This is done. Thanks